src/main/docs/guide/config/environments.adoc

```diff
@@ -16,6 +16,13 @@ $ java -Dmicronaut.environments=foo,bar -jar myapp.jar
 
 The above activates environments called `foo` and `bar`.
 
+To specify configuration via command line:
+[source,bash]
+----
+$ ./gradlew run --args="-endpoints.health.enabled=true -micronaut.environments=test"
```

Comment:
This isn't correct because `micronaut.environments` is not a configuration property. It's only read as a system property. 

The maven example will achieve the desired result, however isn't exactly the same thing as the gradle example because the maven one is just setting system properties. The gradle example will not result in a system property for `endpoints.health.enabled`.

The maven example should be using https://micronaut-projects.github.io/micronaut-maven-plugin/latest/run-mojo.html#appArguments

It also probably makes sense to include this at the end of the section titled "Included PropertySource Loaders"

Reply:
Also it's a good idea to add a note about how the args must be passed to `Micronaut.run`

Reply:
Ok, @jameskleeh I have pushed changes that *I think* address all your concerns... Please let me know how you normally reference a snippet in more than one place... I see the snippets tags, and I tried that and an include, but it doesn't seem to work for me

Reply:
fixed